crates/solidity/testing/snapshots/cst_output/InterfaceDefinition/sample_counter/generated/0.4.11.yml

```diff
@@ -15,44 +15,44 @@ Tree:
   - InterfaceDefinition (Rule): # 0..172 "interface ICounter {\n    // returns the current co..."
       - Interface (Token): "interface" # 0..9
       - Identifier (Token): "ICounter" # 10..18
-      - Group: # 19..172 "{\n    // returns the current count\n    function co..."
+      - _ANON (Rule): # 19..172 "{\n    // returns the current count\n    function co..."
           - OpenBrace (Token): "{" # 19..20
-          - Group: # 25..170 "// returns the current count\n    function count() ..."
-              - ContractBodyElement (Rule): # 25..104 "// returns the current count\n    function count() ..."
-                  - FunctionDefinition (Rule): # 25..104 "// returns the current count\n    function count() ..."
+          - _ANON (Rule): # 58..170 "function count() external view returns (uint);\n\n  ..."
```

Comment:
looks like this node's span didn't include the leading trivia (changed from 25 to 58). Is this intended/a bug?

Reply:
Same in `crates/solidity/testing/snapshots/cst_output/VersionPragma/with_trivia/generated/0.4.11.yml`

Reply:
Yes, a bug